Explorer/Assets/DCL/Multiplayer/Connections/Messaging/Hubs/MessagePipesHub.cs

```diff
@@ -0,0 +1,22 @@
+using DCL.Multiplayer.Connections.Messaging.Pipe;
+
+namespace DCL.Multiplayer.Connections.Messaging.Hubs
+{
+    public class MessagePipesHub : IMutableMessagePipesHub
+    {
+        private readonly InteriorMessagePipe scenePipe = new ();
+        private readonly InteriorMessagePipe islandPipe = new ();
+
+        public IMessagePipe ScenePipe() =>
+            scenePipe;
+
+        public IMessagePipe IslandPipe() =>
+            islandPipe;
```

Comment:
Should this class implement `IMessagePipesHub` interface then as well?

Reply:
IMutableMessagePipesHub derives from IMessagePipesHub as well